apps/wallet/src/ui/app/components/active-coins-card/index.tsx

```diff
@@ -29,38 +27,61 @@ function ActiveCoinsCard({
     const intl = useIntl();
     const aggregateBalances = useAppSelector(accountAggregateBalancesSelector);
 
+    const allCoins = useMemo(
+        () =>
+            Object.keys(aggregateBalances).map((aType) => {
```

Comment:
Use `Object.entries` instead so that you don't need to index into `aggregateBalances` later in this map.

Reply:
Done